iocore/net/UnixNetAccept.cc

```diff
@@ -470,6 +470,7 @@ NetAccept::acceptFastEvent(int event, void *ep)
     } else {
       close_UnixNetVConnection(vc, e->ethread);
     }
+    vc = nullptr;
```

Comment:
> if the accept_till_done is set and we have got a netvc in the last loop, and the accept4() failed (res < 0), and the NetAccept is cancelled.

So, in this situation, `!action_->cancelled` is `false` and `close_UnixNetVConnection(vc, e->ethread)` is called ?
If so, I think this line should be in the above `else` block. Because it looks like vc will not be freed in `Lerror` when `!action_->cancelled` is `true`.

Reply:
All right, I understand what you fixed.
I misunderstood that the `vc->free()` in the [L483](https://github.com/oknet/trafficserver/blob/065c4a25f2b4c308127d08df0af65e0fc5a1cb19/iocore/net/UnixNetAccept.cc#L483) should be called in some conditions. But the vc will be freed when the action is not cancelled.